tests/baselines/reference/smartSelection_JSDocTags5.baseline

```diff
@@ -0,0 +1,30 @@
+/**
+ * @callback Foo
+ * @param {string} data
+ * @param {/**/number} [index]
+ * @return {boolean}
+ */
+
+/** @type {Foo} */
+const foo = s => !(s.length % 2);
+
+
+           number         
+
+
+          {number}        
+
+
+   @param {number} [index]
+ *•                 
```

Comment:
This one looks a little weird (there are some others like it in other tests too) 🤔

Reply:
Yep, I think, need to determine what to choose for each JSDoc tag.